src/components/common/UploadDocs.js

```diff
@@ -117,12 +117,12 @@ export const UploadDocs = ({
   useEffect(() => {
     console.log({ fileList, filePreviews }, 'useEffect');
```

Comment:
I think in Engineering guides there was a comment about not pushing console logs to production.

Pls ignore if I am mistaken

Reply:
I didn't remove that one since it looks like that piece of code is being worked on. I did leave one in myself though, and it is related to another card and provides useful, formatted information in the console.